src/services_models/reserves.jl

```diff
@@ -5,33 +5,30 @@ struct StepwiseCostReserve <: AbstractReservesFormulation end
 """
 This function add the variables for reserves to the model
 """
-function activeservice_variables!(
-    psi_container::PSIContainer,
-    service::SR,
-    contributing_devices::Vector{<:PSY.Device},
-) where {SR <: PSY.Reserve}
-    add_variable(
-        psi_container,
-        [device for device ∈ contributing_devices if PSY.get_available(device)],
-        variable_name(PSY.get_name(service), SR),
-        false;
-        lb_value = d -> 0,
+function make_variable_inputs(
+    ::Type{ServiceVariable},
+    ::PSIContainer,
+    service::T,
+) where {T <: PSY.Reserve}
+    inputs = AddVariableInputs(;
+        variable_name = make_variable_name(PSY.get_name(service), T),
+        binary = false,
+        lb_value_func = x -> 0,
+        devices_filter_func = x -> PSY.get_available(x),
     )
-    return
+    return inputs
```

Comment:
do we want the return at the top to be consistent other functions?

Reply:
Yes.  I'll change it.